oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -484,11 +482,15 @@ private Map<String, String> buildDimensions(
             LOG.info(
                     "No mfa method to set. User is either authenticated or signing in from a low level service");
         }
-        VectorOfTrust vtr = clientSession.getVtrWithLowestCredentialTrustLevel();
-        var mfaRequired = !vtr.getCredentialTrustLevel().equals(CredentialTrustLevel.LOW_LEVEL);
+        var orderedVtrList = VectorOfTrust.orderVtrList(clientSession.getVtrList());
+        var mfaRequired =
+                !orderedVtrList
+                        .get(0)
+                        .getCredentialTrustLevel()
+                        .equals(CredentialTrustLevel.LOW_LEVEL);
         var levelOfConfidence = LevelOfConfidence.NONE.getValue();
-        if (vtr.containsLevelOfConfidence()) {
-            levelOfConfidence = vtr.getLevelOfConfidence().getValue();
+        if (orderedVtrList.get(0).containsLevelOfConfidence()) {
+            levelOfConfidence = orderedVtrList.get(0).getLevelOfConfidence().getValue();
```

Comment:
This is where we spoke about the comma concatenated list (I got the right PR this time!)

Reply:
And I put it in there... but in the wrong PR 😭

Reply:
Actually going to leave it in the later PR as I put the extra methods in there for stringifying - want to keep this to just moving `getLowestCredentialTrustLevel` to simplify it. Lmk if you think it would better fit here